lms/djangoapps/mobile_api/models.py

```diff
@@ -25,3 +23,51 @@ def get_video_profiles(cls):
         Get the list of profiles in priority order when requesting from VAL
         """
         return [profile.strip() for profile in cls.current().video_profiles.split(",") if profile]
+
+
+class AppVersionConfig(ConfigurationModel):  # pylint: disable=model-missing-unicode
+    """
+    Configuration for mobile app versions available.
+    """
+    IOS = "ios"
+    ANDROID = "android"
+    PLATFORM = (
+        (IOS, "iOS"),
+        (ANDROID, "Android"),
+    )
+    KEY_FIELDS = ('platform', 'version')  # combination of mobile platform and version is unique
+    platform = CharField(max_length=50, choices=PLATFORM, blank=False)
+    version = CharField(max_length=50, blank=False)
```

Comment:
If we add some help text, will it show in the admin? I am thinking something like: "Version should be in the format X.X.X where X is a number".